source/examples/rapids-snowflake-cudf/notebook.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<p>I would make this note a warning admonition. We might also want to link to the issue so that readers can check on the progress.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>